starter/cloudrun-graphql/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
+services:
+  postgres:
+    image: postgres:14
```

Comment:
I think we can upgrade here to 15, don't we? cc @smoliji

Reply:
Yes pls.

Reply:
Updated ✅